roles/build_openstack_packages/tasks/serve_gating_repo.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Install python-psutil
+  become: true
+  ansible.builtin.package:
+    name: python-psutil
+
+- name: Getting process IDs of the python http server
+  community.general.pids:
+    pattern: "python -m http.server"
```

Comment:
haven't played with that before, but wondering can we make it more specific like include the port number here, so we can be sure to only kill the one we are interested in?

```
pattern: "python -m http.server 8766"

```

Reply:
Done